Geometry/HGCalCommonData/python/testHGCalV13XML_cfi.py

```diff
@@ -51,6 +54,7 @@
         'Geometry/MuonCommonData/data/mb4Shield/2021/v1/mb4Shield.xml',
         'Geometry/MuonCommonData/data/muonYoke/2021/v3/muonYoke.xml',
         'Geometry/MuonCommonData/data/csc/2021/v2/csc.xml',
+        'Geometry/MuonCommonData/data/mfshield/2017/v2/mfshield.xml',
```

Comment:
@bsunanda it looks like this commit reverted the muon shield changes.

Reply:
Yes - because I am working on a better revised python script. I am looking into overlaps more critically.

Reply:
I thought this was propagating the fix from #30579?

Reply:
yes - but the workflow in #30579 needs further fix. I shall send the fix once my 2 other PR's get merged.

Reply:
The relevant PR is #30654. Once that is merged. I can modify this python script. I thought I shall not wait for that for this PR to go through. I need parts of the PR for some other future work and the python script will follow later on.